apps/judicial-system/web/src/routes/Shared/CaseTable/CaseTable.tsx

```diff
@@ -38,9 +40,16 @@ const hasCellValue = (cell: CaseTableCell): boolean => {
   return cell.value !== null && cell.value !== undefined
 }
 
+const compareString = (
+  a: string | undefined | null,
```

Comment:
NIT: We can also have a shorter syntax since we allow both `undefined` and `null`

```
const compareString = (
  a?: string,
  b?: string,
) => {
...
```

Reply:
I would rather not as the ? may lead to us forgetting to pass an argument to the function. The original forces you to call the function with two arguments. Besides `a?: string` is not enough, it must be `a?:string|null`.